flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/SchedulerTestingUtils.java

```diff
@@ -77,6 +94,47 @@
 
 	private static final long DEFAULT_CHECKPOINT_TIMEOUT_MS = 10 * 60 * 1000;
 
+	private static final Time DEFAULT_TIMEOUT = Time.seconds(300);
+
+	private SchedulerTestingUtils() {}
+
+	public static DefaultSchedulerBuilder newSchedulerBuilder(final JobGraph jobGraph) {
+		return new DefaultSchedulerBuilder(jobGraph);
+	}
+
+	public static DefaultSchedulerBuilder newSchedulerBuilderWithDefaultSlotAllocator(
```

Comment:
I cannot find real usages of these `newSchedulerBuilderWithDefaultSlotAllocator()` and `createScheduler()`. Can you explain why they are needed?

Reply:
It will be used when we change some tests to create DefaultScheduler, one example can be found in `DefaultSchedulerBatchSchedulingTest#createScheduler()` in #10858 .
Another example is `ExecutionGraphSuspendTest` which currently create ExecutionGraph and with a given slotProvider.